src/vcpkg-test/binarycaching.cpp

```diff
@@ -25,13 +25,13 @@ struct KnowNothingBinaryProvider : IBinaryProvider
 
     virtual void push_success(const VcpkgPaths&, const Dependencies::InstallPlanAction&) const override { }
     virtual void prefetch(const VcpkgPaths&,
-                          const std::vector<Dependencies::InstallPlanAction>&,
-                          const std::vector<CacheStatus*>&) const override
+                          View<const Dependencies::InstallPlanAction>,
+                          View<CacheStatus* const>) const override
```

Comment:
```suggestion
                          View<Dependencies::InstallPlanAction>,
                          View<CacheStatus*>) const override
```
`View<T>` implies const (it is just `Span<const T>`)